internals/generators/organization/config.yml.hbs

```diff
@@ -422,6 +422,10 @@ client:
             en: "password"
           label:
             en: "password"
+        remember_me:
+          type: "checkbox"
+          label:
+            en: "Remember Me"
```

Comment:
let's change this to "Automatically log me in to the WiFi service at the next visit" which I hope most end-users will be able to understand